src/DependencyInjection/Compiler/EventHandlerCompiler.php

```diff
@@ -15,7 +16,7 @@ final class EventHandlerCompiler implements CompilerPassInterface
      */
     public function process(ContainerBuilder $container): void
     {
-        $manager = $container->getDefinition('shapecode_twig_template_event.handler_manager');
+        $manager = $container->getDefinition(HandlerManager::class);
```

Comment:
I would suggest to changed this to "findDefinition" and use the Interface.
```suggestion
        $manager = $container->findDefinition(HandlerManagerInterface::class);
```